applications/post-editing.md

```diff
@@ -0,0 +1,23 @@
+---
+description: Human editing of machine translation output
+---
+
+# Post-editing
+
+Human **post-editing** is the review and correction of machine translation output. When post-editing, the human translator checks the original input and the machine translation output, and fixes errors in the output.
+
+Post-editing started to replace human translation because it is faster and cheaper than the traditional human translation process.
+
+Over the 2010s, with rising machine translation quality, it became the dominant human translation workflow.
+
+### Flavours
+With **light post-editing**, the goal is that readers understand the idea of a text. Light post-editing provides the gist of a message. It is usually aimed at internal communications.
+
+With **full post-editing**, the goal is to match human quality. Clients may request full post-editing services for publishable information.
+
+### Metrics
+**Post-editing distance** is a measure of how much the human translator post-edited the machine translation. It's both a metric for a specific translation, and also used in aggregate.
+
+It can be length-normalized and given as a percentage.
+
+**Post-editing effort** is an estimate of how much from the machine translation output needs to be post-edited and how much can be approved as is.
```

Comment:
This needs to be contrasted with PED.

In my book, the main difference is that this one is about *time*, not just Levenshtein.  It's more ideal, it's what we actually want, but harder to measure (so I like how you said "estimate" not "measure") and not well defined.

> **Post-editing effort** is an estimate of how much time the human translator spends post-editing a machine translation.